.github/workflows/codeguru-reviewer.yml

```diff
@@ -0,0 +1,48 @@
+
+# Runs CodeGuru Reviewer on push and pull_request events,
+# and uploads recommendations to the GitHub Security tab.
+# For information on the setup see:
+# https://github.com/aws-samples/aws-codeguru-reviewer-cicd-cdk-sample
+
+name: Analyze with CodeGuru Reviewer
+on: [pull_request, push]
+permissions:
+    id-token: write
+    contents: read
+    security-events: write 
+
+jobs:
+  CodeGuruReviewerScan:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Compile
+      run: ./gradlew compileJava -x test
+    - name: Copy build artifacts
+      run: |
+        mkdir -p class-files
+        cp -r smithy-*/build/classes/java/main/* class-files 
+
+    - name: Assume IAM Role
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        role-to-assume: arn:aws:iam::048169001733:role/GuruGitHubCICDRole
+        aws-region: us-west-2
+
+    - name: Run CodeGuru Reviewer
+      uses: aws-actions/codeguru-reviewer@v1.1
+      continue-on-error: false
+      with:          
+        s3_bucket: codeguru-reviewer-github-profiler-demo-048169001733-uw2
```

Comment:
Where does this bucket come from, and who owns it?

Reply:
It's owned by the CodeGuru team and was created using this Stack:
https://github.com/aws-samples/aws-codeguru-reviewer-cicd-cdk-sample